samples/kvsWebRTCClientMaster.c

```diff
@@ -197,6 +197,7 @@ PVOID sendVideoPackets(PVOID args)
     frame.presentationTs = 0;
     startTime = GETTIME();
     lastFrameTime = startTime;
+    periodicIceTime = GETTIME();
```

Comment:
periodicIceTime doesnt seem to be used anywhere.

Reply:
Yeah. I am modifying the samples to poll the stats periodically. So the current changes in samples will go.

Reply:
Samples modified. Fixed.